src/pages/namespace/Explorer/utils.ts

```diff
@@ -7,6 +10,27 @@ import { stringify as jsonToPrettyYamlStringify } from "json-to-pretty-yaml";
 export const jsonToYaml = (json: Record<string, unknown>) =>
   Object.keys(json).length === 0 ? "" : jsonToPrettyYamlStringify(json);
 
+export const yamlToJsonOrNull = (yaml: string) => {
+  let json;
+  try {
+    json = yamljs.load(yaml);
+  } catch (e) {
+    json = null;
+  }
+  return json;
+};
+
+/**
+ * Compares two yaml strings. Please note that this function will make a
+ * structural comparison, so it will return true if the yaml strings are
+ * structurally equal, even if the order of the keys is different.
+ */
+export const compareYamlStructure = (aYaml: string, bYaml: string) => {
+  const previewJson = yamlToJsonOrNull(aYaml);
+  const fileContentFromServerJson = yamlToJsonOrNull(bYaml);
+  return isEqual(previewJson, fileContentFromServerJson);
+};
```

Comment:
```suggestion
export const compareYamlStructure = (aYaml: string, bYaml: string) => {
  return isEqual(yamlToJsonOrNull(aYaml), yamlToJsonOrNull(bYaml));
};
```

Any two files could be compared, so it doesn't make sense in my opinion to create named variables at this point (or if we do, the names should be more generic).

Reply:
Yes, of course. That was a refactoring artifact. I fixed just fixed that (https://github.com/direktiv/direktiv-ui/pull/552/commits/9c056d091c440a353d21560478d85efe76c708de)